src/components/Integrations/Shopify/Shopify.js

```diff
@@ -99,6 +81,31 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
     </>
   );
 
+  const getData = useCallback(async () => {
```

Comment:
Very minor, but I am not confortable with the name, `getData` seems to be the name of a method to get information, in general that kind of methods returns a value and do not have side effects. 

In place of that, this method has side effects (updating the state) and do not return nothing.

What do you think about `updateData`, `update`, `pull` or `refresh`?

Reply:
Fixed since we removed the function.